packages/prosemirror-lwdita/src/plugin.ts

```diff
@@ -249,7 +250,14 @@ export interface Additions {
   after?: MenuElement[][];
   end?: MenuElement[][];
 }
-
+function isDebugActive(state: EditorState): boolean {
```

Comment:
Rename DebugPlugin - to StructurePlugin or similar and update names in the code.